libraries/features/dma/noimpl/bsg_manycore_dma.cpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * Write memory out to manycore DRAM via DMA
+ * @param[in]  mc     A manycore instance initialized with hb_mc_manycore_init()
+ * @param[in]  npa    A valid hb_mc_npa_t - must be an L2 cache coordinate
+ * @param[in]  data   A buffer to be written out manycore hardware
+ * @param[in]  sz     The number of bytes to write to manycore hardware
+ * @return HB_MC_FAIL if an error occured. HB_MC_SUCCESS otherwise.
+ */
+int hb_mc_dma_write(hb_mc_manycore_t *mc,
```

Comment:
weak?

Reply:
Yeah, I see value in doing that. I'll do it, and add a comment explaining that design decision.

Reply:
Done